src/components/sections/events/index.tsx

```diff
@@ -90,8 +90,7 @@ export const Events: React.FC<EventsProps> = ({ events }) => {
                         </p>
                         <IconButton
                             className="events-slider-link"
-                            href={event.telegram}
-                            target="_blank"
+                            href={`events/${event.slug}`}
```

Comment:
Just for stability and reusability on different pages, otherwise it will be a relative path

```suggestion
                            href={`/events/${event.slug}`}
```